api-catalog-package/src/main/resources/bin/start.sh

```diff
@@ -151,6 +151,23 @@ truststore_pass="${ZWE_configs_certificate_truststore_password:-${ZWE_zowe_certi
 keystore_location="${ZWE_configs_certificate_keystore_file:-${ZWE_zowe_certificate_keystore_file}}"
 truststore_location="${ZWE_configs_certificate_truststore_file:-${ZWE_zowe_certificate_truststore_file}}"
 
+# Check for Java version and set --add-opens Java option in case the version is 17 or later
+java_home="${1:-${JAVA_HOME}}"
+java_version=$("${java_home}/bin/java" -version 2>&1)
+java_version_short=$(echo "${java_version}" | grep ^"java version" | sed -e "s/java version //g"| sed -e "s/\"//g")
+if [[ $java_version_short == "" ]]; then
+    java_version_short=$(echo "${java_version}" | grep ^"openjdk version" | sed -e "s/openjdk version //g"| sed -e "s/\"//g")
+fi
+java_major_version=$(echo "${java_version_short}" | cut -d '.' -f 1)
+ADD_OPENS=""
```

Comment:
There's an implementation that is less verbose if you want to consider it:
Get the class file version from a file within the jvm implementation:
```
JAVA_VERSION=$(${JAVA_HOME}/bin/javap -verbose java.lang.String \
    | grep "major version" \
    | cut -d " " -f5)
```
Then verify the major version:

```
if [ $JAVA_VERSION -ge 61 ]; then
    # This is Java 17
elif [ $JAVA_VERSION -ge 55 ]; then
    # This is Java 11
elif [ $JAVA_VERSION -ge 52 ]; then
    # This is Java 8
fi
```

Reply:
Ok thanks, I can change it